fuel-database/src/tables/resource.rs

```diff
@@ -212,6 +214,8 @@ impl<'a> AssetQuery<'a> {
     }
 }
 
+// TODO: Move definition of `ResourceId` and `Resource` to `fuel-core-interfaces`
```

Comment:
It will be done in the follow-up PRs mentioned in the description. I will create them after the merge of this PR and all comments. If we forget about it, it is not critical todo to forget =) It means that no one uses resources.